src/services/mcp/McpHub.ts

```diff
@@ -33,6 +33,7 @@ import ReconnectingEventSource from "reconnecting-eventsource"
 import * as vscode from "vscode"
 import { z } from "zod"
 import { HostProvider } from "@/hosts/host-provider"
+import { TelemetryService } from "@/services/posthog/telemetry/TelemetryService"
```

Comment:
```suggestion
import { telemetryService } from "@/services/posthog/PostHogClientProvider"
```

I think we can just use `telemetryService` directly from this import without needing to pass it into the constructor

Reply:
We should be abstracting this and not using "PostHog" directly.  This allows flexibility if and when we need to switch to another provider.